tools/src/main/scala/scala/scalanative/build/Config.scala

```diff
@@ -94,50 +145,54 @@ sealed trait Config {
     }
   }
 
-  private[scalanative] lazy val targetsMac = Platform.isMac ||
+  private[scalanative] lazy val targetsMac: Boolean = Platform.isMac ||
     compilerConfig.targetTriple.exists { customTriple =>
       Seq("mac", "apple", "darwin").exists(customTriple.contains(_))
     }
 }
 
+/** Factory to create [[#empty]] [[Config]] objects */
 object Config {
 
-  /** Default empty config object where all of the fields are left blank. */
+  /** Default empty config object where all of the fields are left blank or the
+   *  default value.
+   */
   def empty: Config =
     Impl(
-      nativelib = Paths.get(""),
-      mainClass = None,
-      classPath = Seq.empty,
       basedir = Paths.get(""),
       testConfig = false,
+      moduleName = "",
+      mainClass = None,
+      classPath = Seq.empty,
       logger = Logger.default,
       compilerConfig = NativeConfig.empty
     )
 
   private final case class Impl(
-      nativelib: Path,
-      mainClass: Option[String],
-      classPath: Seq[Path],
       basedir: Path,
       testConfig: Boolean,
+      moduleName: String,
+      mainClass: Option[String],
+      classPath: Seq[Path],
       logger: Logger,
       compilerConfig: NativeConfig
   ) extends Config {
-    def withNativelib(value: Path): Config =
-      copy(nativelib = value)
-
-    def withMainClass(value: String): Config =
-      copy(mainClass = Option(value).filter(_.nonEmpty))
-
-    def withClassPath(value: Seq[Path]): Config =
-      copy(classPath = value)
```

Comment:
Spurious changed: changed order of defintions

Reply:
It now matches the order above for the fields.